src/patterns/gen-ai/aws-rag-appsync-stepfn-opensearch/index.ts

```diff
@@ -534,6 +551,34 @@ export class RagAppsyncStepfnOpensearch extends Construct {
       ],
     }));
 
+    // Minimum permissions for a Lambda function to execute while accessing a resource within a VPC
+    s3_transformer_job_function_role.addToPolicy(new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      actions: [
+        'rekognition:CompareFaces',
+        'rekognition:DetectFaces',
+        'rekognition:DetectLabels',
+        'rekognition:ListFaces',
+        'rekognition:SearchFaces',
+        'rekognition:SearchFacesByImage',
+        'rekognition:DetectText',
+        'rekognition:GetCelebrityInfo',
+        'rekognition:RecognizeCelebrities',
```

Comment:
do we need all the actions listed here ?

Reply:
done -af690096d987c0b91f0b51401b9d8639c21b1ed5